computer-use-server/owui_sync.py

```
@@ -0,0 +1,117 @@
+                {
+                    "filename": file_path.name,
+                    "file_id": entry["file_id"],
+                    "url": entry["url"],
+                }
+            )
+            continue
+
+        try:
+            file_id, url = _upload_file(owui_url, owui_api_key, file_path)
+        except Exception:
+            continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)